test/Reflection/ParameterReflectionTest.php

```diff
@@ -58,6 +58,20 @@ public function testTypeReturn($param, $type)
         self::assertEquals($type, $parameter->detectType());
     }
 
+    /**
+     * This test covers
+     *
+     * @dataProvider paramType2
+     */
+    public function testTypeReturn2(string $param, string $type): void
```

Comment:
Is there a more meaningful way to name this test?

Reply:
```suggestion
    public function testTypeReturnWithNotAllParamsDeclared(string $param, string $type): void
```